src/options.js

```diff
@@ -217,7 +217,17 @@ export function maybeColumn(source) {
 }
 
 export function labelof(value, defaultValue) {
-  return typeof value === "string" ? value : value && value.label !== undefined ? value.label : defaultValue;
+  return typeof value === "string"
+    ? value
+    : typeof value === "function" && value.name && !/^f?[xy]\d*$/.test(value.name)
```

Comment:
Here I’m trying to avoid inheriting a channel name (_e.g._, `fy: (d) => …`). But this doesn’t work for the _fill_ channel, for example.